portal-test-integration/src/com/liferay/portal/lar/test/BaseStagedModelDataHandlerTestCase.java

```diff
@@ -149,6 +149,59 @@ public void testCleanStagedModelDataHandler() throws Exception {
 			portletDataContext, exportedStagedModel);
 	}
 
+	@Test
+	public void testExportImportWithDefaultData() throws Exception {
+
+		// Export default data
+
+		initExport();
+
+		Map<String, List<StagedModel>> dependentStagedModelsMap =
+			addDefaultDependentStagedModelsMap(stagingGroup);
+
+		StagedModel stagedModel = addDefaultStagedModel(
+			stagingGroup, dependentStagedModelsMap);
+
+		if (stagedModel == null) {
+			return;
+		}
+
+		StagedModelDataHandlerUtil.exportStagedModel(
+			portletDataContext, stagedModel);
+
+		validateExport(
+			portletDataContext, stagedModel, dependentStagedModelsMap);
+
+		// Add default data to live site
+
+		Map<String, List<StagedModel>> secondDependentStagedModelsMap =
+			addDefaultDependentStagedModelsMap(liveGroup);
+
+		addDefaultStagedModel(liveGroup, secondDependentStagedModelsMap);
+
+		// Import
+
+		initImport();
+
+		StagedModel exportedStagedModel = readExportedStagedModel(stagedModel);
+
+		Assert.assertNotNull(exportedStagedModel);
+
+		StagedModelDataHandlerUtil.importStagedModel(
+			portletDataContext, exportedStagedModel);
+
+		// Import again
```

Comment:
@matethurzo True, I think even without this the test captures the main use case. I added that in so it could also check a slightly more complex use case.

Specifically, when an entity is imported with a name collision, if the `StagedModelDataHandler` chooses to create a new entity with a slightly changed name (like in the case of WikiNodes), then on the second import, the entity would already exist but with a different name. I thought this made for an interesting case that could break, so I threw it in just to make the test more rigorous.

However, it isn't totally necessary for the use case our ticket is concerned with, so it could also be removed I think.